pkg/web/templates/components/beneficiary_information.html

```diff
@@ -20,6 +20,16 @@ <h2 class="my-4 font-bold">2. BENEFICIARY INFORMATION</h2>
       <input type="text" id="benf_last_name" name="benf_last_name" placeholder="Enter beneficiary's last or family name" class="input-style" />
     </div>
   </div>
+  <div class="grid gap-6 my-4 md:grid-cols-2">
+    <div>
+      <label for="bf_id_dob" class="label-style">Date of Birth (MM/DD/YYYY)</label>
```

Comment:
Let me know if users should insert their birthdate in a different format.

Reply:
Can we use `<input type="date">` so it's localized to the user? 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date

The IVMS101 needs the date in the format YYYY-MM-DD

https://github.com/trisacrypto/trisa/blob/main/proto/ivms101/ivms101.proto#L257C5-L257C26